JS/Lesson26/savas/edit-post-folder/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Posts</title>
+  <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <h1>Posts</h1>
+    <hr>
+    <h3>Edit The Post "<span id="edited-post"></span>"</h3>
+    <div id="form-container" class="posts-container">
+        <form id="edit-form">
```

Comment:
I finally noticed the issue with this page. You need to select the form element in JS, not the button. 
When you select the form element, then you can use "submit" event type for all possible ways of submitting a form. When you select the button, you need to use "onclick" event instead and then cover other methods that can be used to submit the form. See the comment in JS code as well for how to fix the current issue.

Reply:
Wow, the codes are quickly getting very complex and I'm getting confused about what I'm doing where :)